src/Routes/TrustTabs/TrustTabs.scss

```diff
@@ -0,0 +1,5 @@
+@import "~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_all.scss";
+
+.ins-c-dashboard-link {
+    margin: var(--pf-global--spacer--lg) 0;
```

Comment:
Define `margin: var(--pf-global--spacer--lg) 0;` as a global scss var in App.scss and use it here and for `ins-c-title`?

Reply:
is this cuz maybe we'll be using the same thing elsewhere? 🤔

Reply:
yeah that's what I was thinking 🤷‍♂️ no need if you don't think we gonna be though 😁

Reply:
well we already use it twice (button n title spacing), so down for it! great idea 🤗